src/yea/runner.py

```diff
@@ -50,7 +46,8 @@ def prepare(self) -> None:
     def clean(self) -> None:
         os.environ.pop("YEA_RUN_COVERAGE", None)
         if self._tmpdir.exists():
-            shutil.rmtree(self._tmpdir)
+            pass
+            # shutil.rmtree(self._tmpdir)
```

Comment:
remove commented out code?

Reply:
oh.. thanks.. that code needed to be added back. it was removed for debugging.  (eventually it should have a flag for it)